### PR TITLE
feat: Add my claims view

### PR DESCRIPTION
**Pull request by Jonatan-Chaverri**
## Checklist

- [ x ] The title of the PR is clear and descriptive.
- [ ] All tests have been run and passed.
- [ x ] Code follows project conventions (Biome formatting, linting).
- [ ] Documentation has been updated as needed.
- [ x ] Commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) style.

## Type of Change

- [ ] Bugfix
- [ x ] Feature
- [ ] Refactor
- [ ] Chore
- [ ] Documentation
- [ ] Test

### Automated Tests
- [ ] Unit tests added or updated
- [ ] Integration tests added or updated
- [ ] E2E tests added or updated

## Additional Notes

Closes https://github.com/Vagabonds-Labs/cofiblocks/issues/42

Screenshot 2024-11-06 at 6 51 22 PM
Screenshot 2024-11-06 at 6 51 32 PM

### CODE REVIEW

File: apps/web/src/app/user/my-claims/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	status: string;
+	roast: string;
+	type: string;
+	quantity: string;
+	delivery: string;
+	totalPrice: string;
+	address?: string;
+};
+
+export default function MySaleDetails() {
```
Comment (Jonatan-Chaverri) [inline apps/web/src/app/user/my-claims/[id]/page.tsx:19]: I copy this class from [my sales](https://github.com/Vagabonds-Labs/cofiblocks/blob/main/apps/web/src/app/user/my-sales/%5Bid%5D/page.tsx) just changing the status to delivered... not sure if any other change is needed here. I noticed that in the Figma there is an extra option for "Talk with the seller", not sure how to mock that, just show the option and do nothing else?

File: apps/web/src/app/user/my-claims/page.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+				...orderGroup,
+				items: orderGroup.items.filter((item) => item.claimed),
+			}))
+			.filter((orderGroup) => orderGroup.items.length > 0);
+
+		setClaimedOrders(claimedOrders);
+	}, []);
+
+	const openFiltersModal = () => {
+		setIsFiltersModalOpen(true);
```
Comment (Jonatan-Chaverri) [inline apps/web/src/app/user/my-claims/page.tsx:123]: This is doing nothing at the moment, let me know if it should do something

Comment (brolag) [reply apps/web/src/app/user/my-claims/page.tsx:123]: @muguika do we need this filter here?

image

Comment (muguika) [reply apps/web/src/app/user/my-claims/page.tsx:123]: not vital, no!

**Pull request merged**